be/src/storage/rowset/page_io.cpp

```diff
@@ -50,11 +50,15 @@ Status PageIO::compress_page_body(const BlockCompressionCodec* codec, double min
         return Status::OK();
     }
     if (codec != nullptr && uncompressed_size > 0) {
-        compressed_body->resize(codec->max_compressed_len(uncompressed_size));
-        Slice compressed_slice(*compressed_body);
-        RETURN_IF_ERROR(codec->compress(body, &compressed_slice));
-        compressed_body->resize(compressed_slice.get_size());
-
+        if (codec->type() == CompressionTypePB::LZ4_FRAME || codec->type() == CompressionTypePB::ZSTD) {
```

Comment:
1. the compression type in storage is CompressionTypePB::LZ4. Because the input size is 64k, the LZ4_FRAME is not used.
```
storage/rowset/parsed_page.cpp
CompressionTypePB type = CompressionTypePB::LZ4;
```

2. But the `storage/rowset/bitmap_index_writer.cpp` use the LZ4_FRAME, you can change it to LZ4. And also check the read logic right or not.

Reply:
it seems that the default compression type is LZ4F

Reply:
<img width="951" alt="image" src="https://user-images.githubusercontent.com/31178013/175848050-793d3c88-068f-434c-b14b-8865754efbb8.png">